tests/wpt/mozilla/tests/mozilla/calc.html

```diff
@@ -129,7 +129,7 @@
 });
 
 var otherProperties = [
-  ['border-width', 'calc(1px)', '1px 1px 1px 1px'],
+  ['border-width', 'calc(1px)', '1px'],
```

Comment:
This change I guess it's fine to land, but I've opened an issue with this test because it doesn't match what other browsers do at all.